ambf_ros_modules/ambf_comm/scripts/ambf_client.py

```diff
@@ -80,7 +80,9 @@ def __init__(self, client_name='ambf_client'):
         pass
 
     def create_objs_from_rostopics(self):
-        rospy.init_node(self._client_name)
+        
```

Comment:
Can you explain the logic here? Do we randomly assign a new name if a node with existing name exists?

Reply:
It checks if there is already a node running in the script. if there is there is one running it does not create a new one, which would cause it to crash.

Reply:
Oh I see, I was thinking about the other issue which was that in case someone was running two instances of python client as two separate processes and they did not specify a different client_name explicitly then the second instance would force the first instance to close because of ROS name collision.

Reply:
Also, can you update the ambf_addon submodule in the ambf_utilites before updating this PR as otherwise it would take the submodule pointing to a really old version.